### ISSUE TITLE
Ability to preview changes

### ISSUE THREAD

**Issue by svenvanderburg** [research-software-directory/research-software-directory#759]
When changing a page in the RSD it would be very useful to preview how the page will look when published.

Right now the approach of editing is:
* edit the page
* wait for the change to take affect
* see that something is not rendered the way you expected
* repeat
It would be cool to shorten that cycle.

**Comment by jspaaks** [comment]
I agree, that would be great, but this won't be implemented any time soon. Zero-resources problem.

**Comment by svenvanderburg** [comment]
Cool, thanks for the fairness :)
